core/storage/certificate_assessment/gae_models_test.py

```
@@ -188,3 +190,187 @@ def test_commit_log_entry_creation_lifecycle(self) -> None:
         self.assertIsNotNone(log_entry)
         self.assertEqual(log_entry.offering_id, offering.id)
         self.assertEqual(log_entry.commit_type, 'create')
+
+
+class CertificateAssessmentAttemptModelUnitTests(test_utils.GenericTestBase):
+    """Test the CertificateAssessmentAttemptModel class."""
+
+    def _get_sample_attempt_data(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_get_sample_attempt_data`

(ANN202)